server/llm-email-script.py

```
@@ -0,0 +1,124 @@
+import json
+from datetime import datetime
+
+# Get data from previous steps
+analysis_data = previous_step_results.get("LLM Content Analysis", {}).get("result", {})
```

Ruff (0.12.2)

Undefined name `previous_step_results`

(F821)